diff --git a/qt/aqt/main.py b/qt/aqt/main.py
index 3604165..6889559 100644
--- a/qt/aqt/main.py
+++ b/qt/aqt/main.py
@@ -1101,6 +1101,9 @@ title="%s" %s>%s</button>""" % (
     ##########################################################################
 
     def setupAutoUpdate(self) -> None:
+        # Don't check for the latest versions since versions are managed by
+        # Nixpkgs
+        return
         import aqt.update
 
         self.autoUpdate = aqt.update.LatestVersionFinder(self)
